[Docs] Further explained the mythical `InvalidPacket` (#828)
ArchipelagoMW/Archipelago: docs/network protocol.md

### InvalidPacket
Sent to clients if the server caught a problem with a packet. This only occurs for errors that are explicitly checked for.

#### Arguments
| Name | Type | Notes |
| ---- | ---- | ----- |

> | type | str | The [PacketProblemType](#PacketProblemType) that was detected in the packet |

Reviewer: ```suggestion
| type | str | The [PacketProblemType](#PacketProblemType) that was detected in the packet. |
```

> | original_cmd | Optional[str] | the `cmd` argument of the faulty packet, will be `None` if the `cmd` failed to be parsed |

Reviewer: ```suggestion
| original_cmd | Optional[str] | The `cmd` argument of the faulty packet, will be `None` if the `cmd` failed to be parsed. |
```

> | text | str | A descriptive message of the problem at hand |

Reviewer 2: descriptive like `'text': 'Connect'` ? :D
Maybe we should rework the messages one day to make it mach the protol.md :)

Author: Yeah some are more descriptive than others, but I write it down as the way i think it was meant to be

> | text | str | A descriptive message of the problem at hand |

Reviewer: ```suggestion
| text | str | A descriptive message of the problem at hand. |
```

Reviewer 2: ^ that's not a complete sentence (which is why i suggested removing the 3 words on the front)

##### PacketProblemType

> PacketProblemType indicates the type problem that was detected in the faulty packet, the known problem types are below but others maybe added in the future

Reviewer 2: ```suggestion
PacketProblemType indicates the type problem that was detected in the faulty packet, the known problem types are below but others may be added in the future.
```

> PacketProblemType indicates the type problem that was detected in the faulty packet, the known problem types are below but others may be added in the future.

Reviewer: ```suggestion
`PacketProblemType` indicates the type of problem that was detected in the faulty packet, the known problem types are below but others may be added in the future.
```

| Type | Notes |
| ---- | ----- |

> | cmd | `cmd` argument of the faulty packet could not be parsed correctly |

Reviewer: ```suggestion
| cmd | `cmd` argument of the faulty packet that could not be parsed correctly. |
```

> | arguments | arguments of the faulty packet are not correct |

Reviewer: ```suggestion
| arguments | Arguments of the faulty packet which were not correct. |
```

> | cmd | signaling that the `cmd` argument of the faulty packet could not be parsed correctly |
> | arguments | signaling that the arguments of the faulty packet are not correct |

Reviewer 2: If there is a reason to not make this change, let me know and i will approve as is
```suggestion
| cmd | `cmd` argument of the faulty packet could not be parsed correctly |
| arguments | arguments of the faulty packet are not correct |
```

### Retrieved

> Sent to clients as a response the a [Get](#Get) package

Reviewer: ```suggestion
Sent to clients as a response the a [Get](#Get) package.
```

Yeah, I know this isn't your change. But I'm suggesting it anyway.
